license-maven-plugin/src/main/java/com/mycila/maven/plugin/license/util/FileUtils.java

```diff
@@ -133,4 +133,10 @@ public static void copyFilesToFolder(File src, File dst) {
       }
     });
   }
+
+  public static boolean isSubfolder(File subfolder, File folder) {
+    String subfolderPath = subfolder.getAbsolutePath();
+    String folderPath = folder.getAbsolutePath();
+    return subfolderPath.startsWith(folderPath) && subfolderPath.length() > folderPath.length() && subfolderPath.charAt(folderPath.length()) == File.separatorChar;
+  }
```

Comment:
This could be moved to a one-liner with the java Path api. 
Any reason to not use it ?